v2/googlecloud-to-splunk/src/main/java/com/google/cloud/teleport/v2/coders/package-info.java

```diff
@@ -0,0 +1,2 @@
+/** Google Cloud to Splunk coders for Google Cloud Teleport. */
```

Comment:
`mvn spotless:apply` doesn't seem to change add the license header for these two files, so I just copied over the header from the `templates` directory's `package-info.json`

Reply:
Ok. For future reference, there's known issues with Spotless being applied to `package-info.java`: https://github.com/diffplug/spotless/issues/532

I think the way I've worked around it before is to just remove the package-level Javadoc, since it is normally unnecessary. Like in this case, the Javadoc doesn't really add anything, but it's also not something that _needs_ to be removed.

Overall, not a big deal, just something to be aware of.